test/revision

```diff
@@ -1 +1 @@
-11453
+13645
```

Comment:
Oh, right, I see. I'm kind of ambivalent about that, actually. Because now the files are from a mix of sources, rather than just one. But I guess the real solution is to automate the update to easily use a waterfall build rather than a local build, or sanitize the files, or make the waterfall update script get the tests from the waterfall directly rather than checking them in. The downside of the latter is that running these tests now becomes dependent on downloading something external, but currently we have that anyway.